bool toggleForward
bool toggleBackward
bool scaleUp
bool scaleDown
int16 motorID
